README.md

```diff
@@ -1,2 +1,6 @@
 # kutter
-Capstone project AWS Cloud Dev Bootcamp
+"kutter" is a completly serverless content management system for photos and news-articles with an API Gateway based REST Interface.
```

Comment:
If you really want to have a nice readme I would also add some setup instructions ;)